modules

```diff
@@ -1,6 +1,6 @@
 GLUON_FEEDS='packages routing gluon'
 
-OPENWRT_REPO=https://git.openwrt.org/openwrt/openwrt.git
+OPENWRT_REPO=https://github.com/openwrt/openwrt.git
```

Comment:
This switch should be explained in the commit message.

Reply:
Done